src/EditorFeatures/TestUtilities/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -169,64 +198,105 @@ private protected async Task CheckResultsAsync(
                 Assert.Equal(hasSuggestionModeItem.Value, completionList.SuggestionModeItem != null);
             }
 
-            if (checkForAbsence)
+            if (expectedResults.Length is 0)
```

Comment:
```suggestion
            if (expectedResults.Length == 0)
```
Kinda odd to see `is` just to check for a single value)